src/cloud-api-adaptor/docker/README.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+
+The `docker` provider simulates a pod VM inside a docker container.
+
+> **Note**: Run the following commands from the root of this repository.
+
+## Prerequisites
+
+- Install `docker` [engine](https://docs.docker.com/engine/install/)
```

Comment:
Maybe we need add a note here, that we need install `docker` on Kubernetes cluster worker node.